Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -300,6 +300,26 @@ public class CarPlayManager: NSObject {
         return muteButton
     }()
     
+    /**
+     The bar button that brings alternative routes selection during navigation.
+     */
+    public lazy var alternativeRoutesButton: CPBarButton = {
+        let altsButton = CPBarButton(type: .text) { (button: CPBarButton) in
+            guard let template = self.carPlayNavigationViewController?.alternativesListTemplate() else {
```

Comment:
Wouldn't it cause a memory leak? I think it'd be better to hold self weakly.

Reply:
I'm also hesitant about making this a public API. Would be probably better to leave it up to user to create it on their own. Some issues might arise when users want to re-use this property in other places and when `CPInterfaceController` has number of templates that is quite high.